iwa-console/AuthenticationConfig.cs

```diff
@@ -35,41 +36,12 @@ namespace iwa_console
     /// </summary>
     public class AuthenticationConfig
     {
-        /// <summary>
```

Comment:
The Authentication Config is now simplified as we use the PublicClientApplicationOptions. It now only contains the code to read the config from the Json file.